penny_university_frontend/src/middleware/api.ts

```diff
@@ -1,7 +1,8 @@
-import { MiddlewareAPI, Dispatch, Middleware, AnyAction } from "redux"
-import { camelizeKeys, decamelizeKeys } from 'humps'
+import {MiddlewareAPI, Dispatch, Middleware, AnyAction} from "redux"
+import {camelizeKeys, decamelizeKeys} from 'humps'
```

Comment:
You're deleting all the pre/post spaces. I don't really care, since it's syntactically equivalent, but it would be nice to settle on one or the other so that there are less red/green lines in diffs.

Reply:
Yeah this is cause of using auto formatting and my IDE isn't set up the same. I'm gonna fix these.